src/common/relayd/relayd.c

```diff
@@ -117,9 +117,63 @@ static int recv_reply(struct lttcomm_relayd_sock *rsock, void *data, size_t size
 error:
 	return ret;
 }
+/*
+ * Starting at 2.11, RELAYD_CREATE_SESSION payload (session_name & hostname)
+ * have no length restriction on the sender side.
+ * Length for both payload is stored in the msg struct. A new dynamic size
+ * payload size is introduced.
+ */
+static int relayd_create_session_2_11(struct lttcomm_relayd_sock *rsock,
+		uint64_t *session_id, char *session_name, char *hostname,
```

Comment:
both `session_name` and `hostname`can be `const` here (`const char *`).

Reply:
No since they are passed to strcpy which expect "char * restrict. Using const  yield a warning regarding the striping of the const qualifier.

I guess you meant that the pointer can be const:   char * const session_name, char * const hostname.

Reply:
Sorry wrong function altogether....

Reply:
No worries, I think I initially added the comment to the wrong function.